### PR TITLE
removing videos

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
Removing a bunch of videos, and removing videos section from left nav.

### Motivation
These videos are outdated and not being viewed.

### Preview link

https://docs-staging.datadoghq.com/cswatt/videos-purge

update: just removing the videos from the nav.

### CODE REVIEW

File: content/en/logs/_index.md [hunk 1]
```diff
@@ -18,8 +18,6 @@ further_reading:
   text: "Introduction to Logs in Datadog"
 ---
 
-{{< vimeo 293195142 >}}
```
Comment (l0k0ms) [inline content/en/logs/_index.md]: We shouldn't remove this video, this one is up to date.

File: content/en/tracing/trace_search_and_analytics/_index.md [hunk 2]
```diff
@@ -5,8 +5,6 @@ aliases:
   - /tracing/visualization/search/
 ---
 
-{{< vimeo 278748711 >}}
```
Comment (l0k0ms) [inline content/en/tracing/trace_search_and_analytics/_index.md]: We shouldn't remove this video, this one is up to date.

File: content/en/watchdog.md [hunk 3]
```diff
@@ -16,8 +16,6 @@ further_reading:
   text: "Collect your traces"
 ---
 
-{{< vimeo 278057125 >}}
```
Comment (l0k0ms) [inline content/en/watchdog.md]: We shouldn't remove this video, this one is up to date-ish.

**Comment by cswatt** [comment]
The problem with these videos isn't that they're outdated, it's that they're marketing material or have a very marketing tone, and shouldn't be in the docs.

**Pull request merged**